docs/network.md

```diff
@@ -272,3 +272,33 @@ For some applications it may be appropriate to provide a custom `NSURLSessionCon
   _bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:launchOptions];
 }
 ```
+
+## Using a custom `OkHttp` client on Android
+
+To create a custom `OkHttp` client we can use a combination of `OkHttpClientProvider` and `OkHttpClientFactory`, creating an `OkHttp` client factory and passing it to React Native.
+
+```java
```

Comment:
Our website is by-lingual Kotlin/Java so you will have to provide implementation for both.

Reply:
Sure, @cortinico! Should I do the same with the `NSURLConfig` example that only has the Obj-C implementation and add the Swift one?

Reply:
Nope sorry. The website is bilingual only for Java/Kotlin